erc/ERCS/erc-7562.md

```diff
@@ -320,14 +322,15 @@ These are storage rules that help to protect the bundler from denial of service:
 
 ### General Reputation rules
 The following reputation rules apply for both staked and unstaked entities.
+All rules apply to all entities unless specified otherwise.
```

Comment:
Why? We don't want to track opsSeen/opsIncluded for unstaked accounts, and IMHO also not for unstaked factories unless we find a reason to.